### PR TITLE
Fixed #24022 -- Deprecated the ssi tag.

### PR DESCRIPTION
**Pull request by prestontimmons**

### CODE REVIEW

File: docs/ref/templates/builtins.txt [hunk 1]
```diff
@@ -979,6 +979,11 @@ this example, the space around ``Hello`` won't be stripped::
 ssi
 ^^^
 
+.. deprecated:: 1.8
+
+    This tag has been deprecated and will be removed in Django 2.0. The
+    ``include`` tag should be used instead.
```
Comment (berkerpeksag) [inline docs/ref/templates/builtins.txt]: \`\`include\`\` -> :ttag:\`include\`

**Comment by berkerpeksag** [comment]
LGTM.

File: docs/ref/templates/builtins.txt (hunk 1 above)
Comment (aaugustin) [inline docs/ref/templates/builtins.txt]: Often the docs don't include a link when the target is just the previous line.

**Comment by aaugustin** [comment]
This needs:

- an entry in the deprecation timeline -- internals/deprecation.txt
- a sentence in the deprecation note for ALLOWED_INCLUDE_ROOTS -- ref/settings.txt

**Comment by prestontimmons** [comment]
Thanks. I added the ttag processor and extra notes.

File: docs/internals/deprecation.txt [hunk 2]
```diff
@@ -125,6 +125,8 @@ details on these changes.
 
 * The ``--list`` option of the ``migrate`` management command will be removed.
 
+* The :ttag:`ssi` template tag will be removed.
```
Comment (MarkusH) [inline docs/internals/deprecation.txt]: Removing the link syntax prevents updates in upcoming releases when the reference isn't available anymore.

Comment (prestontimmons) [inline docs/internals/deprecation.txt]: Okay. I removed the link.

**Comment by timgraham** [comment]
merged in de9ebdd39cb4f4b65475b43e0e32772d5a315654, thanks.

**Pull request closed without merging**